EEBOS/owl-ui
# Chore/refactor [sc-878] build process

### Short summary

- moved non component packages to root level
- theme package being built with just sass and postcss
- made design folder into a package, which the theme imports

---

### Test steps

1. start storybook
2. create component
3. build system
4. start web

All four should run as expected.
